src/geopackage/feature.cpp

```diff
@@ -0,0 +1,97 @@
+#include "GeoPackage.hpp"
+
+geojson::Feature geopackage::build_feature(
+  const sqlite_iter& row,
+  const std::string& geom_col
+)
+{
+    std::vector<double> bounding_box(4);
+    const auto id                    = row.get<std::string>("id");
+    geojson::PropertyMap properties  = build_properties(row, geom_col);
+    geojson::geometry geometry       = build_geometry(row, geom_col, bounding_box);
+
+    // Convert variant type (0-based) to FeatureType
+    const auto wkb_type = geojson::FeatureType(geometry.which() + 1);
+
+    // Points don't have a bounding box, so we can say its bbox is itself
+    if (wkb_type == geojson::FeatureType::Point) {
+        const auto& pt = boost::get<geojson::coordinate_t>(geometry);
+        bounding_box[0] = pt.get<0>();
+        bounding_box[1] = pt.get<1>();
+        bounding_box[2] = pt.get<0>();
+        bounding_box[3] = pt.get<1>();
+    }
+    
+    switch(wkb_type) {
+        case geojson::FeatureType::Point:
```

Comment:
The reason for this was readability, since if I construct the bounding box within the switch, it would look like this (due to variable initialization bypassing):
```
bounding_box[0] = boost::get<geojson::coordinate_t>(geometry).get<0>();
bounding_box[1] = boost::get<geojson::coordinate_t>(geometry).get<1>();
bounding_box[2] = boost::get<geojson::coordinate_t>(geometry).get<0>();
bounding_box[3] = boost::get<geojson::coordinate_t>(geometry).get<1>();
```

In any case, I can do this, but I'm not sure that the loss in readability is worth it?

Reply:
Write an inline function for the logic and put that call in the switch?

Reply:
🤦🏾 Ah duh, that makes sense. I'll include this in the next push, thanks!

Reply:
Implemented in e8f4646.